website/docs/r/notification_destination.html.markdown

```diff
@@ -174,8 +174,40 @@ resource "newrelic_notification_destination" "foo" {
     token  = "u+E8EU3MhsZwLfZ1ic1A"
   }
 }
-``` 
+```
+
+#### Mobile Push
+```hcl
+resource "newrelic_notification_destination" "foo" {
+  account_id = 12345678
+  name = "mobile-push-example"
+  type = "MOBILE_PUSH"
 
+  property {
+    key = "userId"
+    value = "12345678"
+  }
+}
+```
+
+#### [AWS Event Bridge](https://docs.newrelic.com/docs/alerts-applied-intelligence/notifications/notification-integrations/#eventBridge)
+```hcl
+resource "newrelic_notification_destination" "foo" {
+  account_id = 12345678
+  name = "event-bridge-example"
+  type = "EVENT_BRIDGE"
+
+  property {
+    key = "AWSAccountId"
+    value = "123456789123456"
+  }
+
+  property {
+    key = "AWSRegion"
+    value = "us-east-2"
+  }
+}
+```
 
```

Comment:
Adding examples related to slack will be helpful.

Reply:
Slack option will only be available via import, do you think we should add an example for that too?

Reply:
Yes